### ISSUE TITLE
Hyperparamter tuning on FashionMNIST dataset

### ISSUE THREAD

**Issue by AnikaJagati** [DSC-ChitkaraUniv/Hacktoberfest_2021#19]
Hey , I would love to contribute a code on the working of a CNN model on the FashionMNIST dataset and show the tuning using dropout layer , L2 regularisation and keras autotuning , plotting the loss and accuracy graphs, along with added commments for ease of code understanding.

**Comment by bhaveshgoyal182** [comment]
Hey @AnikaJagati ,
Go ahead & proceed with the steps as mentioned in Readme 👍🏻

**Comment by AnikaJagati** [comment]
Hey , I've made a pull request to add the IPYNB  file under the Deep Learning folder.
Have a look and lemme know :)
